cwf/gateway/integ_tests/assertions.go

```diff
@@ -104,12 +105,36 @@ func (tr *TestRunner) DisconnectAndAssertSuccess(imsi string) {
 
 // Query policy usage records from pipelined, and assert that there is no entry
 // for the IMSI. This means no policy flows are installed for the IMSI.
-func (tr *TestRunner) AssertPolicyEnforcementRecordIsNil(imsi string) {
+func (tr *TestRunner) AssertEnforcementRecordIsEmptyForSub(imsi string) {
 	recordsBySubID, err := tr.GetPolicyUsage()
 	assert.NoError(tr.t, err)
 	assert.Empty(tr.t, recordsBySubID[prependIMSIPrefix(imsi)])
 }
 
+func (tr *TestRunner) AssertPolicyEnforcementRecordIsNotNil(recordsBySubID RecordByIMSI, imsi, ruleID string) {
+	record := recordsBySubID[prependIMSIPrefix(imsi)][ruleID]
+	assert.NotNil(tr.t, record, fmt.Sprintf("No enforcement record for %s %s", imsi, ruleID))
+}
+
+func (tr *TestRunner) AssertPolicyEnforcementRecordIsNil(recordsBySubID RecordByIMSI, imsi, ruleID string) {
```

Comment:
Just a suggestion:
What about instead of 
`AssertPolicyEnforcementRecordIsNotNil`
`AssertPolicyEnforcementRecordIsNil`
(see to tell the difference you need to look very close to the name of the function)

we make them more different so we can easily see what they do with the naked eye
`AssertPolicyEnforcementRecordHasTraffic`
`AssertPolicyEnforcementRecordIsEmpty`
or 
`AssertHasTrafficForRecordInPolicyEnforcement`
`AssertEmptyForRecordInPolicyEnforcement`

Reply:
Ok will fix.